samples/boards/cc3220sf_launchxl/echo/README.rst

```diff
@@ -0,0 +1,55 @@
+.. _echo_sample:
+
+Socket Echo Sample over CC3220SF WiFi
+#####################################
+
+Overview
+********
+
+This sample reimplements the networking sockets echo sample, leveraging
+the CC3220SF WiFi host driver which is part of the SimpleLink SDK.
+
+This is based on the Zephyr sockets echo sample, with the following
+differences:
+
+* since SimpleLink's arpa/inet.h is missing inet_ntop, rather than
+  pulling in the Zephyr networking stack just for an IP printing
+  utility function, that bit of code is removed;
```

Comment:
Clarify what "that bit of code" is referring to.

Reply:
yes, done.